src/shared/__tests__/libs/iota/utils.spec.js

```diff
@@ -10,6 +10,18 @@ import {
 } from '../../../libs/iota/utils';
```

Comment:
`isLastTritZero` needs to be imported
```suggestion
    isLastTritZero,
} from '../../../libs/iota/utils';
```